launch/includes/ir.launch.xml

```diff
@@ -8,7 +8,7 @@
   <arg name="ir" />
   
   <!-- Rectified image -->
-  <node pkg="nodelet" type="nodelet" name="rectify_ir"
+  <node pkg="nodelet" type="nodelet" name="$(arg ir)_rectify_ir"
```

Comment:
@130s We already depends on an `arg` in `depth.launch.xml` (https://github.com/ros-drivers/rgbd_launch/blob/indigo-devel/launch/includes/depth.launch.xml#L14).
If `ir` is not given, this launch file is invalid, and if `ir` is set to empty string, processed messages are published as global namespace and nodelet name will be `_rectify_ir`, which is also valid as ROS node name.(but I think it will be unusual name)